emsdk.py

```diff
@@ -2761,6 +2767,10 @@ def print_tools(t):
 
     return 0
   elif cmd == 'construct_env':
+    if len(sys.argv) > 2 and sys.argv[2] == '--csh':
+      global SHELL
+      SHELL = 'csh'
+      del sys.argv[2]
```

Comment:
It looks like we have `extract_bool_arg` to do this kind of thing already.

Reply:
> we have `extract_bool_arg` to do this kind of thing already.

Yes, ok, I didn't see it --- I was not looking for it, to be honest.  The right way to do this is `argparse`, if you want my 2 cents...  :-)